src/main/resources/lang/english.lang

```diff
@@ -1265,6 +1265,16 @@ heal reasons:
 	wither: withering, withered, wither potion, a wither effect, wither effect, wither
 	wither_spawn: wither spawning, wither spawn, wither summoning, a wither summoning, a wither spawn, a wither spawning
 
+# -- Fishing States --
+fishing states:
```

Comment:
I feel like some of these names will sound odd in script lines, but I am also not too sure how to improve them. Anyone have any ideas?